crossing-training.lic

```diff
@@ -979,8 +979,8 @@ class CrossingTraining
 
   def smith(full_name)
     wait_for_script_to_complete('smith', ['bronze', full_name, 'buy'])
-    dispose_trash(right_hand)
-    dispose_trash(left_hand)
+    noun = get_data('recipes')['crafting_recipes'].select { |item| item['name'] == full_name }.first['noun']
```

Comment:
Should this use `recipe_lookup` (from `DRCC`) like smith.lic does?

Reply:
I wasn't aware of recipe_lookup - that sounds like a good idea to me. I'll update this to use that function instead.